src/datasource.js

```diff
@@ -75,6 +76,7 @@ export default class OCIDatasource {
           region: this.defaultRegion,
           tenancyOCID: this.tenancyOCID,
           environment: this.environment,
+          compartmentOCID: this.compartmentOCID,
           datasourceId: this.id
         }
       ],
```

Comment:
Unrelated to this change, there is a bug in the logic below which prevents the result of the datasource test from being properly displayed on the Grafana data source page in the browser. The code should be changed as follows:

```
diff --git a/src/datasource.js b/src/datasource.js
index 05d154a..47e7b31 100644
--- a/src/datasource.js
+++ b/src/datasource.js
@@ -83,12 +83,11 @@ export default class OCIDatasource {
       range: this.timeSrv.timeRange()
     })
       .then((response) => {
-        if (response.status === 200) {
-          return {
-            status: 'success',
-            message: 'Data source is working',
-            title: 'Success'
-          }
+        console.log("testDatasource endpoint response:", response)
+        return {
+          status: 'success',
+          message: 'Data source is working',
+          title: 'Success'
         }
       })
       .catch(() => {
```

Reply:
ok